app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: {maximum: 50}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
+  validates :email, presence: true, length: {maximum: 255},
+  format: {with: VALID_EMAIL_REGEX},
+  uniqueness: {case_sensitive: false}
```

Comment:
```suggestion
  validates :email, presence: true, length: {maximum: 255},
    format: {with: VALID_EMAIL_REGEX},
    uniqueness: {case_sensitive: false}
```


Các giá trị 255 6 50 thì khai báo vào setting file -> tìm hiểu gem config để làm nhé

các khai báo biến trong file setting  thì 
`validations.length.digit_5: 5` ... ri để sau có thể dễ dàng tái sử dụng